nxcompshad/Shadow.cpp

```diff
@@ -36,6 +36,13 @@
 #include "Poller.h"
 #include "Manager.h"
 
+/* On GNU/Hurd, we don't have asm-generic/errno.h (where
+ * EBADFD is defined on Linux systems).
+ */
+#ifndef EBADFD
+# define EBADFD 77
```

Comment:
@sthibaul: @Ionic: I reworked this PR after some chat on #debian-hurd. It seems that we use EBADFD as error code and message sort of arbitrary. In theory, we could use any E*** code here (because we simply use the error code for generating some textual log message).

Thus, I simply switched to EBADF instead of EBADFD (which should be portable).

Agree?

@Ionic: next iteration round for this PR. Thanks!